src/ReinforcementLearningZoo/src/algorithms/nfsp/nfsp.jl

```diff
@@ -0,0 +1,190 @@
+"""
+Neural Fictitious Self-Play (NFSP) agent implemented in Julia.
+
+See the paper https://arxiv.org/abs/1603.01121 for more details.
+"""
+export NFSPAgent, NFSPAgents
+
+using Distributions: TruncatedNormal
+
+mutable struct NFSPAgents <: AbstractPolicy
+    agents::Dict{Any, AbstractPolicy}
+end
+
+mutable struct NFSPAgent <: AbstractPolicy
+    η
+    rng
+    rl_agent::Agent
+    sl_agent::Agent
+end
+
+function initW(out_size, in_size)
```

Comment:
`initW` is so common a name that might cause conflict with other algorithms.

```suggestion
function _NFSP_initW(out_size, in_size)
```

Reply:
Thanks for your reviews and sorry for the late response.

>> I don't think users need to directly use `NFSPAgent`, so no need to export this one.
>
>> We don't use plural case `NFSPAgents` to differentiate with `NFSPAgent`.
> >
> >Since we don't really need to export `NFSPAgent`, we can maybe rename `NFSPAgent` to `NFSPAgentInstance`?

Here, `NFSPAgent` is the core policy to play the game. In my view, agents in one multi-agent experiment may choose different policies to play the game. (maybe just like players can use different schemes in one game) 

And `NFSPAgents` is more like a special [MultiAgentManager](https://github.com/JuliaReinforcementLearning/ReinforcementLearning.jl/blob/639717388fb41199c98b90406bea76232bc6294d/src/ReinforcementLearningCore/src/policies/agents/multi_agent.jl#L8) in which all agents choose `nfsp` policy to learn the best response. For more clarity, I'll extract 'NFSPAgents' and its relative methods to a new file and renamed it as 'NFSPAgentManager'.

>  >You can follow this format here:
>
> > I believe this docstring should go to `NFSPAgents`. Please follow the Julia guideline https://docs.julialang.org/en/v1/manual/documentation/

Thanks for your reminder. I'll supplement the description about `NFSPAgent` and its experiment later.

> `initW` is so common a name that might cause conflict with other algorithms.

Thank you for pointing it out. Maybe I'll rename it as `_TruncatedNormal` because of the used distribution.